web/docs/Backend Starters/Decoupled WordPress/configuring-build-hooks.md

```diff
@@ -0,0 +1,61 @@
+---
+id: 'configuring-build-hooks'
+title: 'Configuring Build Hooks'
+slug: '/backend-starters/decoupled-wordpress/configuring-build-hooks'
+sidebar_position: 5
+---
+
+## Create Your First Build Hook
+
+### What Are Build Hooks?
+
+Build hooks enable certain events in the CMS to trigger builds of a decoupled
+frontend; such as content updates to post, or publishing a new page.
+
+### Install and Activate WP Webhooks Plugin
+
+- Install and activate the
+  [WP-Webhooks Plugin](https://wordpress.org/plugins/wp-webhooks/).
+
+### Creating a Build Hook That Triggers When a Post Is Created Or Updated
+
+1. Navigate to the Pantheon Dashboard
+1. Open the site's settings
+1. Under the **Builds** tab, Click **Add Build Hook**
+1. Name your hook and select a branch to target using the **Branch to Build**
+   dropdown. Copy this URL
```

Comment:
Forgot to mention this on the first round; Do we want to leave this portion for the platform docs? That way we can keep this doc a bit more platform neutral.

marking similar parts with +

Reply:
That is a good point. I agree I think it makes sense to leave the build hook generation to the platform docs.